lib/tasks/manageiq/cypress.rake

```diff
@@ -0,0 +1,16 @@
+# This check ensures that we only load this task under the `app:` prefix when
+# in `manageiq-ui-classic`, and loads it normally under `manageiq`
+#
+# Without it, a few things happen:
+#
+#   - There is an error since there is no manageiq/integration file in the path
+#   - If the above is addressed, it will load this task in two contexts when
+#     loading from manageiq-ui-classic:
+#     - cypress:ui:*
+#     - app:cypress:ui:*
+#
+if Rails.root
+  require "manageiq/integration"
+
+  ManageIQ::Integration::CypressRakeTask.new(:ui)
```

Comment:
You might also noticed that this file got... well way smaller.

The logic for this now lives in `ManageIQ/manageiq` as part of this PR:

https://github.com/ManageIQ/manageiq/pull/20350

I figured if other plugins what to implement `cypress` testing, this should be shared code and easily repeatable.